### ISSUE TITLE
R6 inherited method links assume files that may not exist

### ISSUE THREAD

**Issue by nealrichardson** [r-lib/roxygen2#963]
See https://enpiar.com/arrow-r-site/reference/InputStream.html for example. The "inherited methods" sections link to other reference pages, the pages for the parent classes where you'd think those methods are documented. But they don't exist. They're base classes that aren't really interesting. And even if I had documented them, I could have given the files a different `@name`, so it's not safe to assume that `ClassName.html` will exist.

**Comment by gaborcsardi** [comment]
Right. We'll probably need to have some additional information to know 
1) whether to create those links, and
2) where they should point to.
